### PR TITLE
Registration page

### PR DESCRIPTION
**Pull request by Priyanka7001**

### CODE REVIEW

File: Resource/Page_Object/registrationResource.robot [hunk 1]
```diff
@@ -0,0 +1,72 @@
...
+    Open My Browser
+    Log To Console    visiting registration page
+    Click on My Account Dropdown
+    Sleep    2s
+   
+
+Fill Registration Details
+    [Arguments]    ${name}    ${mobileNumber}    ${password}    ${countryCode}=None
+    Enter User Name    ${name}
+#    Select Country Code    ${countryCode}
```
Comment (ShubhamB2595) [inline Resource/Page_Object/registrationResource.robot]: Remove commented code

File: Test_Suite/testRegistration.robot [hunk 2]
```diff
@@ -0,0 +1,32 @@
...
+*** Test Cases ***
+001 Verify the Functionality of Signup Button from Myaccount Dropdown
+    Visit Registration Page
+    ${actualUrl}=    Get Current Page URL
+    Should Contain     ${actualUrl}    signup   both URL are equal
+
+
+002 Verify Registering an Account with valid User Name and Mobile Number
+    Visit Registration Page
+    Fill Registration Details    Rutika    9021657453    Rutika@123    Rutika@123
```
Comment (ShubhamB2595) [inline Test_Suite/testRegistration.robot]: Move this test data to variable files

File: Test_Suite/testRegistration.robot [hunk 3]
```diff
@@ -0,0 +1,32 @@
...
+    ${pageUrl}=    Get Current Page URL
+    Should Contain    ${pageUrl}    myaccount    both eual
+
+003 Verify Registering an Account with invalid OTP
+    Visit Registration Page
+    Fill Registration Details    Rutika    8007022590   Rutika@123    Rutika@123
```
Comment (ShubhamB2595) [inline Test_Suite/testRegistration.robot]: Move this too

File: selenium-screenshot-1.png
Comment (ShubhamB2595) [inline selenium-screenshot-1.png:1]: Ignore this screenshot and report files 
Add this to hitignore file
Do not push this

File: Test_Suite/testRegistration.robot (hunk 3 above)
Comment (Priyanka7001) [reply Test_Suite/testRegistration.robot]: DOne

**Pull request merged**